microservices-orders/docs/index.rst

```diff
@@ -76,7 +76,7 @@ It is build on the |cp|, including:
 
 * JDBC source connector: reads from a sqlite database that has a table of customers information and writes the data to a Kafka topic, using Connect transforms to add a key to each message
 * Elasticsearch sink connector: pushes data from a Kafka topic to Elasticsearch
-* ksqlDB: another variant of a fraud detection microservice
+* |ksqldb|: another variant of a fraud detection microservice
```

Comment:
@rspurgeon not your error (mine) but could you fix the line above:

`It is build on the |cp|, including:` to `It is built on the |cp|, including:`